crates/bifrost/src/error.rs

```diff
@@ -10,14 +10,14 @@
 
 use std::sync::Arc;
 
+use crate::log_chain_extender::ExtendLogChainError;
+use crate::loglet::OperationError;
```

Comment:
bad bot

Reply:
Sry for that. Fixing it. Maybe I should ditch Rust Rover at some point.